lib/domain/selectors.ts

```diff
@@ -197,6 +198,11 @@ export const recipeSelectors: domainInformationSelector = {
         ingredientsSelector: 'li[class="wprm-recipe-ingredient"]',
         directionsSelector: 'div[class="wprm-recipe-instruction-text"]',
     },
+    'seriouseats.com' : {
```

Comment:
Would you mind alphabetizing this within the array? It helps avoid merge conflicts in prs :slightly_smiling_face: